test/ServerComparison.TestSites/StartupHelloWorld.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Diagnostics;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Runtime;
+
+namespace ServerComparison.TestSites
+{
+    /// <summary>
+    /// To make runtime to load an environment based startup class, specify the environment by the following ways:
+    /// 1. Drop a Microsoft.AspNet.Hosting.ini file in the wwwroot folder
+    /// 2. Add a setting in the ini file named 'ASPNET_ENV' with value of the format 'Startup[EnvironmentName]'. For example: To load a Startup class named
+    /// 'StartupHelloWorld' the value of the env should be 'NtlmAuthentication' (eg. ASPNET_ENV=HelloWorld). Runtime adds a 'Startup' prefix to this and loads 'StartupHelloWorld'.
```

Comment:
Fix all comments (`NtlmAuthentication` -> `Helloworld`)or remove this. I think this is repeated in a number of places.